dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -98,8 +98,10 @@ class DdAndroidGradlePlugin @Inject constructor(
     ) {
         val isObfuscationEnabled = isObfuscationEnabled(variant, datadogExtension)
         val isNativeBuildRequired = variant.isNativeBuildEnabled
+        val isNativeSymbolsTaskRequired =
+            isNativeBuildRequired || datadogExtension.additionalSymbolFilesLocations?.isNotEmpty() == true
 
-        if (isObfuscationEnabled || isNativeBuildRequired) {
+        if (isObfuscationEnabled || isNativeBuildRequired || isNativeSymbolsTaskRequired) {
```

Comment:
seems it can be simplified a bit

```suggestion
        if (isObfuscationEnabled || isNativeSymbolsTaskRequired) {
```

Reply:
let's keep here as it is for better readability